### ISSUE TITLE
Unable to use version 3.0 with Swift Package Manager

### ISSUE THREAD

**Issue by matepapp** [Raureif/WikipediaKit#14]
Hey! 👋 

First of all, thanks for the awesome package! Currently, I'm facing an issue where I couldn't use the latest version (v3.0) with Swift Package Manager (Xcode 12, macOS Big Sur).

Xcode automatically suggests to use 2.0.7
<img width="737" alt="Screenshot 2020-11-28 at 18 00 24" src="https://user-images.githubusercontent.com/17027165/100521411-ca655180-31a3-11eb-98b5-ef9795bfd5ea.png">

and if I explicitly set 3.0.0 as an exact version number
<img width="719" alt="Screenshot 2020-11-28 at 18 01 17" src="https://user-images.githubusercontent.com/17027165/100521427-e832b680-31a3-11eb-9863-2874ff9b03e5.png">

I've also tried to build directly from the `master` branch but then I got a couple of API availability related error messages. Do you have any idea how to solve this problem?

**Comment by frankrausch** [comment]
Hi Mate,

thanks for your kind words!

I think this may be related to the Deployment Targets of your host project. Does it work with a clean Xcode project with Deployment Targets set to iOS 12?

I have also released a fix a minute ago that correctly reflects the supported platforms (v3.0.1).

If I recall correctly, I had to drop support for older OS versions because I use the modern `Result<.success, .failure>` type in the latest code.
